Makefile

```diff
@@ -92,8 +92,7 @@ ut:
 	RUST_BACKTRACE=1 ${CARGO} test $(VIRIOFS_COMMON) --bin nydusd -- --nocapture --test-threads=8
 
 macos-fusedev:
-	# nydus-cached/nydus-image does not support macos yet
-	${CARGO} build --target ${ARCH}-apple-darwin --target-dir ${current_dir}/target-fusedev --features=fusedev --release --bin nydusctl --bin nydusd
+	${CARGO} build --target ${ARCH}-apple-darwin --target-dir ${current_dir}/target-fusedev --features=fusedev --release --bin nydusctl --bin nydusd --bin nydus-image --bin nydus-cached
```

Comment:
How about removing `--bin ... --bin ...`, so that we can build binaries that might be created in the future.

Reply:
blobfs still needs a lot of fixes to build on macos, so those `--bin` args cannot be removed for now.